speechbrain/tokenizers/SentencePiece.py

```diff
@@ -200,35 +200,41 @@ def _csv2text(self):
                 self.annotation_train
                 + " is not a file. please provide annotation file for training."
             )
+        annotation = ",".join(self.annotation_read)
         logger.info(
-            "Extract "
-            + self.annotation_read
-            + " sequences from:"
-            + self.annotation_train
+            "Extract " + annotation + " sequences from:" + self.annotation_train
         )
         annotation_file = open(self.annotation_train, "r")
         reader = csv.reader(annotation_file)
         headers = next(reader, None)
-        if self.annotation_read not in headers:
-            raise ValueError(
-                self.annotation_read + " must exist in:" + self.annotation_train
-            )
-        index_label = headers.index(self.annotation_read)
-        text_file = open(self.text_file, "w+")
-        row_idx = 0
-        for row in reader:
-            if self.num_sequences is not None and row_idx > self.num_sequences:
-                print(
-                    "Using %d sequences to train the tokenizer."
-                    % self.num_sequences
+
```

Comment:
Why is there changes in the SentencePiece file ? Could you remove it from the PR if there is nothing new?

Reply:
Because I use both transcription and translation to train a BPE model (ESPnet use a joint dictionary, I want to replicate their results). They are (transcription/translation) separated into two fields, so, I modified some functions for adding this functionality. Or, could we open another PR on this functionality, just make this one simpler, since this modification will affect all recipes.

Reply:
Yes, we need a separate PR for that. And I am not sure that this is really justified. Can't we do it in another way? The problem is that, currently, we can't take multiple fields to train, right ?

Reply:
Yeah, the issue is that we can't read multiple fields to train. I delete the change of this part, and I will open another pr on this issue. Do you have any suggested way to do it?

Reply:
A short solution would be to do a field that concatenates both fields. Otherwise, it should be backward compatible i.e we should not need to convert all recipes, hence, accept a string or a list of string.